lib/readline.js

```diff
@@ -326,6 +329,30 @@ Interface.prototype.question = function(query, cb) {
   }
 };
 
+Interface.prototype.question[
+  Symbol.for('nodejs.util.promisify.custom')] = function(query) {
```

Comment:
This should use `const { customInspectSymbol } = require('internal/util');` and then `Interface.prototype.question[customInspectSymbol]`

Reply:
will fix!